### PR TITLE
feat(automation): dev to staging

### PR DESCRIPTION
**Pull request by GopherJ**
close: #43  #50

### CODE REVIEW

File: resources/para-200-genesis [hunk 1]
```diff
@@ -1 +1 @@
-0x00000000000000000000000000000000000000000000000000000000000000000018746fe351f855209b4f9aae70942a224b6c5837958ab7391fc7bebc72f6f13f03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c11131400
\ No newline at end of file
+0x00000000000000000000000000000000000000000000000000000000000000000069f0f43fa5c6c2edee2ef3e5d6cbd0a735358afdb1e2c1e4768cecfd431149ea03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c11131400
```
Comment (kaichaosun) [inline resources/para-200-genesis:1]: As mentioned, such files are too heavy to be tracked with Git, I suggest we upload it to a shared google drive, people can download it if needed.

**Review by kaichaosun — APPROVED** [review]
Looks good, only one concern I raised.

File: resources/para-200-genesis (hunk 1 above)
Comment (GopherJ) [reply resources/para-200-genesis:1]: yes I'll do it

**Review by yz89 — CHANGES_REQUESTED** [review]
As discussed earlier, we do not track the contents of the resource directory, such as para-200-genesis, para200.wasm.

**Review by yz89 — APPROVED** [review]

**Comment by GopherJ** [comment]
- remove rococo_local.json, types.json

**Pull request merged**